pkg/api/plugin.go

```diff
@@ -70,48 +68,41 @@ type DeployFn func(context.Context, map[string]interface{}) error
 
 // PluginConfig is passed into NewPlugin
 type PluginConfig struct {
-	SyncImage       string
 	AcceptLanguages []string
-	GenevaConfig    GenevaConfig
+	Config          OverrideConfig
 
 	TestConfig TestConfig
 }
 
+// OverrideConfig is passed to GenerateConfig interface to overwrite default values
+// it is cut-down version of Config structure
```

Comment:
Is this going to end up being the admin api?

Reply:
More like Plugin config bootstrap template config